docs/deployment/functional-layers.md

```diff
@@ -0,0 +1,53 @@
+---
+title: UDS Core Functional Layers
+type: docs
+weight: 9
```

Comment:
Do we think this doc should come before the [flavors deployment](https://github.com/defenseunicorns/uds-core/blob/main/docs/development/flavor-specific-dev.md?plain=1#L4) doc? Another idea is referencing this doc in the main getting started doc for the [doc site.](https://github.com/defenseunicorns/uds-docs-refresh/blob/main/src/content/docs/getting-started/install-and-deploy-uds.md)

Reply:
I intentionally shoved it lower in the priority order and did not directly link it anywhere else since I don't think we want to generally encourage this path (ref [this comment](https://github.com/defenseunicorns/uds-core/issues/868#issuecomment-2401713123)). This ideally should just be a reference "just in case" + to provide the warning about usage.

Reply:
Yeah that's a good reminder, forgot about that. Should probably leave that were it is.